.gitignore

```diff
@@ -1,2 +1 @@
-
-.env
\ No newline at end of file
+web/.env
```

Comment:
@markFalcone please refer to the suggestions I left you after reviewing the first homework, you should include here node_modules in order to not push them to the repository! 

https://github.com/markFalcone/bigcorp-lotteries-api/issues/1